lib/jekyll/commands/serve.rb

```diff
@@ -2,6 +2,21 @@ module Jekyll
   module Commands
     class Serve < Command
       class << self
+        def process(opts)
+          @overrides = opts["config"] if opts["config"]
+
+          opts = configuration_from_options(opts)
+          destination = opts["destination"]
+          setup(destination)
+
+          start_up_webrick(opts, destination)
+        end
+
+        def custom_configs
+          return nil unless @overrides
+          @overrides
```

Comment:
I was under the impression that when we define an `attr_accessor`, its called again via `#initialize` method